docs/en/03_How_tos/redirections.md

```diff
@@ -36,11 +36,18 @@ Once you have that then the redirection should be as follows:
 RewriteCond %{HTTP_HOST} ^my\.domain\.govt\.nz$
 RewriteCond %{HTTPS} !=on
 Rewritecond %{HTTP:X-Forwarded-Proto} !https [NC]
-RewriteRule ^(.*)$ https://%{HTTP_HOST}%{REQUEST_URI} [L,R=301]
+RewriteCond %{REQUEST_URI} ^(?:/public)(.*)$ [NC]
+RewriteRule ^(.*)$ https://%{HTTP_HOST}%1 [L,R=301]
```

Comment:
Hmm, just eyeing the regex over, there could be some problems. Could you test with the following?

* `https://test.cwp.govt.nz/this_should_rewrite`
* `https://test.cwp.govt.nz/public_this_should_not_drop_public`

Reply:
Good point. With the public not being the document root the /public/ prefix is always there, but it's important to have it working for the public folder being the document root as well, which was the whole point.
Just tested the above and I'll update the change slightly to
```
RewriteCond %{REQUEST_URI} ^/(?:public/)?(.*)$ [NC]
RewriteRule ^(.*)$ https://%{HTTP_HOST}/%1 [L,R=301]
```

Reply:
Updated.